pillar.example

```diff
@@ -1,6 +1,7 @@
+---
 exim:
   config:
-    # See http://pkg-exim4.alioth.debian.org/README/update-exim4.conf.8.html for details.
+    # See http://pkg-exim4.alioth.debian.org/README/update-exim4.conf.8.html for details
```

Comment:
While you didn't add this link, it's currently broken, despite what's said on this page:

* https://wiki.debian.org/PkgExim4

Perhaps we should leave this link but put another one beneath it?